### PR TITLE
M3d seg loading

### PR DESCRIPTION
**Pull request by MJHamar**

### CODE REVIEW

File: adapt_med_seg/data/dataset.py [hunk 1]
```diff
@@ -49,54 +50,37 @@ def __init__(
...
-        
+
         if self.train and idx not in getattr(self, "_test_indices", []):
             data_item = self._processor.train_transform(ct_npy, gt_npy)
         else:
             data_item = self._processor.zoom_transform(ct_npy, gt_npy)
 
-        return data_item, gt_npy, modality
+        # squeeze the channel dimension. it's always 1
+        data_item["label"] = data_item["label"].squeeze(0)
```
Comment (m-krastev) [inline adapt_med_seg/data/dataset.py:77]: This is a little concerning but if you say there are no issues with loading ok.

Comment (MJHamar) [reply adapt_med_seg/data/dataset.py:77]: The reason there can only be one is because on line 66, we select exactly one mask. The reason why we need to squeeze here is because SegVol's original logic was to select the class after the data loading happened (something like how we did it), although I have no idea how they trained that

Comment (MJHamar) [reply adapt_med_seg/data/dataset.py:77]: This was the easiest and most bugfree way to do this

File: adapt_med_seg/models/lightning_model.py [hunk 2]
```diff
@@ -42,58 +42,50 @@ def __init__(
...
-                train_labels=train_labels,
-                modality=modality,
-            )
 
-            if loss is None:
-                loss = _loss
-            else:
-                loss = loss + _loss
+        train_organs = task
+        train_labels = gt_label
```
Comment (m-krastev) [inline adapt_med_seg/models/lightning_model.py:62]: Shouldn't there be `to(self.device)` here

Comment (m-krastev) [reply adapt_med_seg/models/lightning_model.py:62]: Nvm, it's there a bit above
